flow/cmd/worker_common.go

```diff
@@ -0,0 +1,17 @@
+package cmd
+
+import (
+	"log"
+
+	"go.temporal.io/sdk/contrib/opentelemetry"
+	"go.temporal.io/sdk/interceptor"
+)
+
+func GetTemporalClientInterceptors() []interceptor.ClientInterceptor {
+	// Maybe do this only when otel is actually enabled?
+	tracingInterceptor, err := opentelemetry.NewTracingInterceptor(opentelemetry.TracerOptions{})
+	if err != nil {
+		log.Printf("failed to create tracing interceptor: %v\n", err)
```

Comment:
return nil instead of slice containing nil to properly ignore

Reply:
my bad, was testing something out